ui/apps/platform/src/Containers/Vulnerabilities/components/SnoozedCveToggleButton.tsx

```diff
@@ -5,11 +5,19 @@ import { SearchFilter } from 'types/search';
 export type SnoozeCveToggleButtonProps = {
     searchFilter: SearchFilter;
     setSearchFilter: (searchFilter: SearchFilter) => void;
+    snoozedCveCount: number | undefined;
 };
 
-function SnoozeCveToggleButton({ searchFilter, setSearchFilter }: SnoozeCveToggleButtonProps) {
+function SnoozeCveToggleButton({
+    searchFilter,
+    setSearchFilter,
+    snoozedCveCount,
+}: SnoozeCveToggleButtonProps) {
     const isSnoozeFilterActive = searchFilter['CVE Snoozed']?.[0] === 'true';
     const buttonText = isSnoozeFilterActive ? 'Show observed CVEs' : 'Show snoozed CVEs';
+    const showCountBadge =
+        snoozedCveCount !== undefined && snoozedCveCount > 0 && !isSnoozeFilterActive;
```

Comment:
Because count is from response, what do you think are pro and con from run-time viewpoint:
* `snoozedCveCount !== undefined`
* `typeof snoozedCveCount === 'number'`

Reply:
Is the concern that even if it is typed `number | undefined`, the response could be other than a number during run-time? I wonder if we could utilize an `ensureNumber` function like we've been doing for Advanced Filters 🤔 @dvail

Reply:
Yes, question, not necessarily a change request.
* Partly to understand consensus about balance between compile-time versus run-time checks.
* Even more, to understand consensus about readability of positive versus double-negative conditions.

Reply:
Yes, `typeof snoozedCveCount === 'number'` would be safer for a runtime check, especially since the result comes from an API response. In theory we could run into trouble if the API endpoint changed out from under us and returned a non-numeric value. I've been thinking of how we can deal with this in a general sense, but I don't have any solution that is perfect at this point. If one even exists.

I can make the change now though since it is trivial and does provide us with a bit more resilience at runtime.